test/index.js

```diff
@@ -43,17 +87,28 @@ describe('cron-allowed-range', function() {
       );
     });
 
+    ['0-59 0-23 1-31 1-11,12 0-6'].forEach(expression => {
+      it(`should not throw any error when the values are exactly on the range'${expression}'`, function() {
```

Comment:
Nit: there should be a space after `range`.

Reply:
Sure will fix this